.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,6 @@
+eclipse.preferences.version=1
```

Comment:
I think this was generated by eclipse and not project-keeper, could you please remove it?